src/kili/mutations/asset/__init__.py

```diff
@@ -93,6 +94,7 @@ def append_many_to_dataset(
                 - For VIDEO projects (and not VIDEO_LEGACY), you can specify a value with key 'processingParameters' to specify the sampling rate (default: 30).
                     Example for one asset: `json_metadata_array = [{'processingParameters': {'framesPlayedPerSecond': 10}}]`.
             disable_tqdm: If `True`, the progress bar will be disabled
+            blocking: If `True`, the function will wait until the assets are imported.
```

Comment:
For me, we could find another name more explicit like `wait_until_sucess`? And add a bit more info on the arg description
Here I doubt that someone without the context of all our discussion would understand when to use this when having a look at the doc